e3/testsuite/__init__.py

```diff
@@ -168,12 +190,15 @@ def testsuite_main(self, args=None):
             help="This is default with this testsuite framework. The option"
             " is kept only to keep backward compatibility of invocation with"
             " former framework (gnatpython.testdriver)")
-        parser.add_argument(
-            "--disable-cleanup",
-            dest="enable_cleanup",
-            action="store_false",
-            default=True,
-            help="disable cleanup of working space")
+
+        if self.dev_mode:
```

Comment:
According to the spec above, this `--disable-cleanup` should NOT exist in dev mode, so I believe this condition should be negated.

Reply:
Good catch, fixed